java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/InputStream.java

```diff
@@ -12,70 +10,18 @@
  *
  * @see OutputStream
  */
-public class InputStream {
-    /**
```

Comment:
There was 20 unused constructors.

Reply:
Yep, it's completely crazy.
At least now they're finally gone!